web/src/components/drawer-diag.tsx

```
@@ -0,0 +1,78 @@
+      </Button>
+      
+      {isOpen && (
+        <div 
+          className="fixed inset-0 z-50 bg-black/50 flex items-center justify-center"
+          onClick={handleClose}
+          onKeyDown={(e) => e.key === 'Enter' && handleClose()}
+          role="dialog"
+          aria-modal="true"
+          aria-labelledby="dialog-title"
+          tabIndex={0}
+        >
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)